### PR TITLE
Create nighty build in prow

### PR DESCRIPTION
**Pull request by Demonsthere**
**Description**

Changes proposed in this pull request:

- New pipeline script for the nightly build

**Related issue(s)**
Implements #454

### CODE REVIEW

File: prow/jobs/kyma/kyma-integration.yaml [hunk 1]
```diff
@@ -223,3 +223,26 @@ periodics:
         requests:
           memory: 1Gi
           cpu: 400m
+- name: kyma-gke-nightly
+  cron: "0 0 * * 1-5" # At 00:00 on every day-of-week from Monday through Friday
+  labels:
+    preset-slack-notifications: "true"
+    <<: *gke_job_labels_template
+  decorate: true
+  extra_refs:
```
Comment (aszecowka) [inline prow/jobs/kyma/kyma-integration.yaml]: this is a periodic job, do we have kyma-project/kyma cloned somehow?

File: prow/scripts/cluster-integration/kyma-gke-nightly.sh [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+shout "Authenticate"
+date
+init
+readonly DNS_DOMAIN="$(gcloud dns managed-zones describe "${CLOUDSDK_DNS_ZONE_NAME}" --format="value(dnsName)")"
+export DNS_DOMAIN
+
+shout "Delete old cluster - gkeint-nightly-${LAST_TIMESTAMP}"
+date
+removeCluster "${LAST_TIMESTAMP}"
```
Comment (aszecowka) [inline prow/scripts/cluster-integration/kyma-gke-nightly.sh]: can this script handle situation that old cluster does not exist?

File: prow/scripts/cluster-integration/kyma-gke-nightly.sh [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
```
Comment (aszecowka) [inline prow/scripts/cluster-integration/kyma-gke-nightly.sh]: I am not sure if this will work in case that periodic job is not executed every day.

Comment (Demonsthere) [reply prow/scripts/cluster-integration/kyma-gke-nightly.sh]: You are right, changing the strategy to a lookup & delete scenario

File: prow/scripts/cluster-integration/kyma-gke-nightly.sh (hunk 2 above)
Comment (Demonsthere) [reply prow/scripts/cluster-integration/kyma-gke-nightly.sh]: Implementing guard to delete clusters only if we detect them

File: prow/jobs/kyma/kyma-integration.yaml (hunk 1 above)
Comment (Demonsthere) [reply prow/jobs/kyma/kyma-integration.yaml]: My copy&paste error, adding kyma refs :)

**Pull request merged**